packages/nc-gui/composables/useIntegrationsStore.ts

```
@@ -461,19 +461,19 @@ const [useProvideIntegrationViewStore, _useIntegrationStore] = useInjectionState
       }
     }[]
 
-    dynamicIntegrations.sort((a, b) => (a.meta.order ?? Infinity) - (b.meta.order ?? Infinity))
+    dynamicIntegrations.sort((a, b) => (a.manifest.order ?? Infinity) - (b.manifest.order ?? Infinity))
 
     for (const di of dynamicIntegrations) {
       let icon: FunctionalComponent<SVGAttributes, {}, any, {}> | VNode
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)